config/config.go

```diff
@@ -0,0 +1,40 @@
+package config
+
+import "os"
+
+type conf struct {
+	getEnv func(string) string
+}
+
+func newConfig() *conf {
+	return &conf{
+		getEnv: func(key string) string { return os.Getenv(key) },
+	}
+}
+
+func (c *conf) getConf() Config {
+	switch c.getRuntimeEnvironment() {
+	case envNameProd:
+		return c.getProdConfig()
+	default:
```

Comment:
Is there value in having a case for `envNameDev` and then in the default still return dev, but do a warning log? or panic /error perhaps?

Reply:
Having looked through more, if we assume the behaviour of `getRuntimeEnvironment` doesn't change, then it would never not be dev. Happy either way

Reply:
Makes sense, let's make it panic since this should never happen